bin/integration_test.sh

```diff
@@ -56,6 +64,10 @@ docker-compose up -d antenna
 # from inside the containers.
 ./bin/wait.sh localhost 8888
 
+# Remove anything in the bucket that Antenna generated at startup so we have an
+# empty bucket to work with.
+docker-compose run -u "${HOSTUSER}" test ./bin/aws_s3.sh rm "${DESTBUCKET}" --recursive
```

Comment:
This fixes an edge cases where running `bin/integration_test.sh` right after building things but before starting any of the services caused the test to fail.